src/test/java/rife/engine/TestEngine.java

```diff
@@ -44,6 +44,25 @@ public void setup() {
         }
     }
 
+    @Test
+    void testTomcatPlain()
```

Comment:
I would appreciate a few more tests to ensure that fundamental items like path info, headers, cookies, binary responses, content type, ... and more are working properly.

Reply:
I have [another branch](https://github.com/ethauvin/rife2/tree/localhost-tests-fix) with more test and also fixes the 127.0.0.1 hard-coding issue. I'll PR it when this PR is merged.